File: 杨荣建小建在线考试系统/考试任务/net操作题/MemoSystem/Model/memoInfo.cs

using System;
namespace Model.Model
{
	public partial class memoInfo
	{
		public memoInfo()
		{}
		#region Model
		private int _memoid;
		private string _memotitle;
		private string _memocontent;
		private DateTime? _begintime;
		private DateTime? _endtime;
		private int? _userid;
		/// <summary>
		/// 
		/// </summary>
		public int memoId
		{
			set{ _memoid=value;}
			get{return _memoid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string memoTitle
		{
			set{ _memotitle=value;}
			get{return _memotitle;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string memoContent
		{
			set{ _memocontent=value;}
			get{return _memocontent;}
		}
		/// <summary>
		/// 
		/// </summary>
		public DateTime? beginTime
		{
			set{ _begintime=value;}
			get{return _begintime;}
		}
		/// <summary>
		/// 
		/// </summary>
		public DateTime? endTime
		{
			set{ _endtime=value;}
			get{return _endtime;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int? userId
		{
			set{ _userid=value;}
			get{return _userid;}
		}
		#endregion Model

	}
}

File: 杨荣建小建在线考试系统/考试任务/java操作题_sqlserver/memoSystem/src/com/shop/po/Memoinfo.java
package com.shop.po;

import java.util.Date;
import java.util.List;

import javax.validation.constraints.Size;

public class Memoinfo {
    private Integer memoId;
    
    public Integer getMemoId() {
        return memoId;
    }

    public void setUserid(Integer memoId) {
        this.memoId = memoId;
    }
    
    private String memoTitle;
    
    public String getMemoTitle() {
        return memoTitle;
    }

    public void setMemoTitle(String memoTitle) {
        this.memoTitle = memoTitle;
    }
    
private String memoContent;
    
    public String getMemoContent() {
        return memoContent;
    }

    public void setMemoContent(String memoContent) {
        this.memoContent = memoContent;
    }
    
    
    
private Date beginTime;
    
    public Date getBeginTime() {
        return beginTime;
    }

    public void setBeginTime(Date beginTime) {
        this.beginTime = beginTime;
    }
    
private Date endTime;
    
    public Date getEndTime() {
        return endTime;
    }

    public void setEndTime(Date endTime) {
        this.endTime = endTime;
    }
    
private Integer userUserId;
    
    public Integer getUserId() {
        return userUserId;
    }

    public void setUserId(Integer userUserId) {
        this.userUserId = userUserId;
    }
}